core-services/image-mirroring/dptp-internal/mapping_dptp_internal_ci

```diff
@@ -0,0 +1,55 @@
+registry.svc.ci.openshift.org/ci/clonerefs@sha256:5fa2e37b4216742eae0924511c9f13bac03fb3a3688a2e2d28e92f18009237a7=registry.ci.openshift.org/ci/clonerefs:test
```

Comment:
I do not think we need those images for ci.
IMHO, if we need an image for CI, they should be backed up by something we can rebuild from.

Reply:
Removing this.
Rehearsing.